analytical/utils.py

```diff
@@ -84,6 +84,8 @@ def get_identity(context, prefix=None, identity_func=None, user=None):
             if user is None:
                 user = get_user_from_context(context)
             if get_user_is_authenticated(user):
+                identity_func = getattr(settings, 'ANALYTICAL_IDENTIFY_FUNC',
+                                        identity_func)
                 if identity_func is not None:
                     return identity_func(user)
                 else:
```

Comment:
I know this is not part of the PR, but if you wouldn't mind: While you're here touching the code, I'd like to revert the `if`-clause to reduce cognitive overhead, i.e.

```python
                if identity_func is None:
                    return user.get_username()
                else:
                    return identity_func(user)
```
Feel free to ignore this request.